tools/remove_dependencies.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
+
+# Copyright 2019 The Vitess Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Remove tools and dependencies installed by "make tools"
+
+get_arch() {
+  uname -m
+}
+
+function fail() {
+	echo "ERROR: ${1}"
+	exit 1
+}
+
```

Comment:
There's no need to call `name` and `uname -m` in so many places. We can set globals here:
PLATFORM=$(uname -s)
ARCH=$(uname -m)